src/config.rs

```diff
@@ -22,6 +24,7 @@ impl Default for Config {
             broadcast: broadcaster::Config::default(),
             evm_chain_configs: vec![],
             tofnd_config: TofndConfig::default(),
+            private_key: [0; 32],
```

Comment:
```suggestion
            private_key: ECDSASigningKey,
```
Wanna try deserializing into the struct directly?